interfaces/apparmor/template.go

```diff
@@ -135,6 +136,8 @@ var defaultTemplate = `
   /{,usr/}bin/cpio ixr,
   /{,usr/}bin/cut ixr,
   /{,usr/}bin/date ixr,
+  /{,usr/}bin/dbus-daemon ixr,
+  /{,usr/}bin/dbus-run-session ixr,
```

Comment:
When do we want snaps to run those?

Reply:
Please see the individual commit message for this: https://github.com/snapcore/snapd/pull/5968/commits/af9edc528d94ce5fd33eb2eff3713b029c083303

They can run them any time they want to create an dbus private bus that is private to the snap. This isn't conceptually different from them setting up an abstract socket without dbus or simply shipping their own dbus-daemon and dbus-run-session. We could yank these two rules (but leave the others) so they must ship these, but I didn't see a benefit to that.